templates/README.example.md

```diff
@@ -36,7 +36,7 @@ bin/check
 
 This runs
 
-* keyword check (console.log, TODO, puts, ...)
+* keyword check (console.log, T0D0, puts, ...)
```

Comment:
yeah, let's talk about it face to face. it can be removed for certain projects if needed, but our definition of done includes "no todos in our code", so we should automate it if possible (actually, sometimes it also annoyed me [e.g. when i wanted to commit the readme file], but other times, it saved my commit)